### PR TITLE
User class

### LINKED ISSUE #29: Create a class-representation of a User.

### PR DESCRIPTION
**Pull request by Puh00**
Co-authored-by: OlofSjogren <olofsjogren97@gmail.com>

Closes #29

### CODE REVIEW

**Comment by OscarSanner** [comment]
Looking good!

**Review by OscarSanner — COMMENTED** [review]
Everything looks solid

File: app/src/androidTest/java/com/goayo/debtify/model/UserTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.goayo.debtify.model;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
```
Comment (yenanw) [inline app/src/androidTest/java/com/goayo/debtify/model/UserTest.java:6]: Nice with test class!

**Review by gabrielbrattgard — COMMENTED** [review]
Looking good and solid. I approve the pull request. (TEST)

**Pull request merged**